src/client/pages/RetroCertsCertificationPage/index.js

```diff
@@ -75,8 +75,14 @@ function RetroCertsCertificationPage(props) {
     }
   }
 
-  const weekSeekWorkPlan =
-    userData.seekWorkPlan[userData.weeksToCertify.indexOf(weekIndex)];
+  // Most (99.99%) users have the same seekWorkPlan for all weeks in
+  // weeksToCertify, in which case their seekWorkPlan is a one element array.
+  // The rest have an array whose length matches the weeksToCertify length.
+  const seekWorkPlanIndex =
+    userData.seekWorkPlan.length === 1
+      ? 0
+      : userData.weeksToCertify.indexOf(weekIndex);
```

Comment:
Oh, I think you could also do weekForUser - 1 for seekWorkPlanIndex, but the old code works as well.

Reply:
Do you mean: `userData.seekWorkPlan.length === 1 ? 0 : weekForUser - 1`?

Reply:
Yes you do, good idea thanks!

Reply:
Yes, sorry, didn't see this.